gammapy/estimators/points/core.py

```diff
@@ -268,7 +268,6 @@ def write(
             sed_type = self.sed_type_init
         table = self.to_table(sed_type=sed_type, format=format)
 
-        # TODO: rather ugly - better method?
```

Comment:
@adonath this TODO was written when the code was different, and then it was updated but the TODO was never removed. Do you think it is still ugly?